README.md

```diff
@@ -5,34 +5,164 @@
 
 Micro-service supporting VPN activities
 
-## Developer docs
+## Endpoints
 
-### Pre-requisites
+### AIC - Affiliate Identifier Cookie
 
-* Rust - https://www.rust-lang.org/tools/install
+`/aic`:
+- POST only
+- Accepts: JSON data with `flow_id`, `cj_id`
+- Returns: JSON data with  `aic_id`, `expires` (a timestamp)
+- A cookie should then be set with the stated expiration time and the returned `aic_id`
+- Success - 201
 
-Coming soon:
-* Postgres
+`/aic/<aicID>` endpoint:
```

Comment:
I think it would be worth explicitly stating here that this endpoint will accept any valid uuid, even if we don't know about it in advance. I imagine there is a functional requirement for this feature, but it goes against what I would normally expect from a REST-style PUT endpoint (i.e. update only, no create).

Reply:
I don't know why i thought this was the right thing to do. I'm changing it so it 404s for unknown aic_ids.

Reply:
fa8ca70